extensions/wikia/CuratedContent/maintenance/SpecialPromoteToCuratedContent.php

```diff
@@ -0,0 +1,163 @@
+<?php
+/**
+ * Script to upload image and description from Special:Promote to Curated Content
+ * Usage: run in context of a given wiki
+ * SERVER_ID=wiki_id php SpecialPromoteToCuratedContentMigrator.php
+ *
+ * @ingroup Maintenance
+ */
+
+require_once( dirname( __FILE__ ) . '/../../../../maintenance/Maintenance.php' );
+
+class SpecialPromoteToCuratedContentMigrator extends Maintenance {
+	public function __construct() {
+		$this->addOption( 'dry-run', 'Dry run - do not save changes to curated content' );
+		parent::__construct();
+	}
+
+	public function execute() {
+		global $wgCityId, $wgLang, $wgDBname;
+
+		$dryRun = $this->hasOption('dry-run');
+
+		$change = false;
+		$cv = new CityVisualization();
+		$communityDataService = new CommunityDataService( $wgCityId );
+
+		$curatedContentData = [
```

Comment:
Yep, I can use that - it'd be useful to have that in dev though not to introduce artificial dependency between those two stories